analysis/controller_test.go

```diff
@@ -159,11 +159,6 @@ func (f *fixture) run(analysisRunName string) {
 	f.runController(analysisRunName, true, false, c, i, k8sI)
 }
 
-func (f *fixture) runExpectError(analysisRunName string, startInformers bool) { //nolint:unused
```

Comment:
Why do we remove this test?

Reply:
It's not being used.

Reply:
I see this comment... //nolint:unused 

Maybe it's not being used but it should be present. Let me readd it just in case.

Reply:
Done